src/resmom_win/start_exec.c

```diff
@@ -2258,10 +2230,18 @@ finish_exec(job *pjob)
 			}
 			si.hStdInput = hReadPipe_dummy;
 		}
-		if (script_out != -1)
+		if (script_out != -1) {
```

Comment:
line 2620. is error logging for similar failed case not required?

Reply:
That one is to close a valid handle. If in case it is already invalid we don't need to do anything since we log it above itself.